src/core/i18n/translator.js

```diff
@@ -11,13 +13,42 @@ export default class Translator {
     return this._interpolate(stringToInterpolate, interpolationParams);
   }
 
+  /**
+   * If translations is json, parse it and choose the correct plural form.
+   * Otherwise just return the non-interpolated translation string.
+   * @param {string} translations
+   * @param {number} count
+   * @returns {string}
+   */
   static _selectPluralization (translations, count) {
```

Comment:
Maybe this could be renamed to _selectStringToInterpolate since the selection of the plural form is happening inside _selectPluralForm

Reply:
sure, updated!